x-pack/legacy/plugins/apm/public/context/UrlParamsContext/resolveUrlParams.ts

```diff
@@ -73,7 +73,7 @@ export function resolveUrlParams(location: Location, state: TimeUrlParams) {
     detailTab: toString(detailTab),
     flyoutDetailTab: toString(flyoutDetailTab),
     spanId: toNumber(spanId),
-    kuery: kuery && decodeURIComponent(kuery),
+    kuery: kuery && legacyDecodeURIComponent(kuery),
```

Comment:
what about encoding, should it change as well?

Reply:
i think we'll want to have the kuery param normalize to the default encoding, which `legacyDecodeURIComponent` also handles.